doc/lua_api.txt

```diff
@@ -5288,6 +5288,16 @@ Call these functions only at load time!
 * `minetest.register_on_item_eat(function(hp_change, replace_with_item, itemstack, user, pointed_thing))`
     * Called when an item is eaten, by `minetest.item_eat`
     * Return `itemstack` to cancel the default item eat response (i.e.: hp increase).
+* `minetest.register_on_item_pickup(function(itemstack, picker, obj, time_from_last_punch,  ...))`
+    * Called when an item is picked up, after `itemdef.on_pickup`.
+    * Function is added to `minetest.registered_on_item_pickups`.
+    * Oldest functions are called first.
+    * Parameters:
+      * `itemstack`: The item to be picked up. Do not modify!
```

Comment:
Also here: copy the stack for safety.

Reply:
We never copy arguments for each registered function when calling callbacks with `core.run_callbacks`. So, I've removed the note here (not modifying arguments should be the default anyway). I've also added a copy once for all invocations.
A part of the other callbacks run in builtin always copy and don't use `core.run_callbacks`. I could do that here too if wanted.